modules/test/playwright/utils/onExit.ts

```diff
@@ -34,7 +34,7 @@ function handleSignal(signal: number): void {
 	}
 
 	if (signal !== -1) {
-		process.exit(128 + SIGNALS[signal]);
+		process.exit(128 + Object.values(SIGNALS)[signal]);
```

Comment:
This code was copied from https://github.com/liferay/liferay-frontend-projects/blob/970a0ba34f064360d83661f560e60b792a6f22e0/projects/npm-tools/packages/npm-scripts/src/utils/SignalHandler.js.

And according to Node's documentation, the code is correct:

- https://nodejs.org/docs/latest-v20.x/api/process.html#signal-events
- https://nodejs.org/docs/latest-v20.x/api/process.html#event-exit

Additionally, the change doesn't make sense IINM because:

1. `SIGNALS` is an object
2. `Object.values(SIGNALS)` is an array which looks like `[1, 2, 3, 15, -1]`
3. The `signal` argument (according to Node's documentation) is a `string` which, when used to index the array, is going to return `undefined`.

So I guess the solution to this problem would be declaring the `signal` argument as `string`, right?